src/firetower/settings.py

```diff
@@ -212,10 +212,20 @@ def dev_default(key: str, default: str = "") -> str:
     # Pagination
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
     "PAGE_SIZE": 50,
-    # Authentication
+    # Authentication - use our custom class that reads from IAP middleware.
+    # This bridges Django middleware user to DRF's Request.user property.
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "firetower.auth.authentication.IAPAuthentication",
+    ],
+    # Permissions
     "DEFAULT_PERMISSION_CLASSES": [
         "firetower.incidents.permissions.IsAuthenticated",
     ],
+    # Only accept JSON - rejects form-encoded requests which bypass CORS preflight.
+    # Defense in depth alongside browser third-party cookie blocking and CORS.
+    "DEFAULT_PARSER_CLASSES": [
+        "rest_framework.parsers.JSONParser",
```

Comment:
This doesn't break the admin page, right?

Reply:
doesn't break it, but I noticed the CSRF exemption was applying to admin views when it should not, so I've fixed that.